### DISCUSSION TITLE
Add functionality

### DISCUSSION THREAD

**Discussion by alibabaedge** [BitCannaGlobal/WeedMobile#84] [Ideas]
Hey guys, it would be great to have faceID/touchID function for unlock the wallet.

**Comment by kjincgroup** [comment]
A great to have function which boost the convenience of using the mobile app.

Majority of wallet mobile app has provided Face ID unlock function, like Coinbase wallet

**Comment by alibabaedge** [comment]
Also would be great to have an indication of current using chain, for now I'm not sure which chain (testnet or mainnet I use).

**Comment by TJ-BitCanna** [comment]
Thanks for bringing it to the attention!

Marked it is a potential enhancement.
